doc/tui_rstgen.py

```diff
@@ -0,0 +1,306 @@
+"""Provides a module for generating Fluent TUI .rst files."""
```

Comment:
```suggestion
"""Provides a module for generating Fluent TUI result files."""
```
Use result like was done for in the previous file.

Reply:
Done. Thank you.